packages/core/src/2d/sprite/SpriteRenderer.ts

```diff
@@ -440,10 +473,12 @@ export class SpriteRenderer extends Renderer {
 enum SpriteRendererUpdateFlags {
   /** UV. */
   UV = 0x2,
-  /** WorldVolume and UV . */
-  RenderData = 0x3,
+  /** Color. */
+  Color = 0x4,
   /** Automatic Size. */
-  AutomaticSize = 0x4,
+  AutomaticSize = 0x8,
+  /** WorldVolume and UV. */
+  WorldVolumeAndUV = RendererUpdateFlags.WorldVolume | SpriteRendererUpdateFlags.UV,
```

Comment:
Should be `The number of vertices changes`, so we need update `position`, `uv` and `color`.